parser/src/parser.cpp

```diff
@@ -204,16 +213,32 @@ int main(int argc, char* argv[])
     return 1;
   }
 
-  //--- Check workspace and project directory ---//
+  //--- Check database and project directory existence ---//
+
+  bool isNewDb = cc::util::connectDatabase(
+    vm["database"].as<std::string>(), false) ? false : true;
+  bool isNewProject = !checkProjectDir(vm);
+
+  if (isNewProject ^ isNewDb)
+  {
+    LOG(error)
+      << "Database and working directory existence are inconsistent. Use -f for reparsing!";
```

Comment:
are -> is

Specify '--force' to do a clean parse!

(Yet again, trying to eliminate the *reparse* phrase from here.)

Reply:
I just copied the "*Use -f for reparsing!*" part from other error messages present in the project, to be consistent with them. Again, I think this name collision issue regarding the *reparse* phrase should not be part of this pull request.